plugin/web/extensions/sidebar/core/components/content/common/DatasetCard/Field/Fields/index.ts

```diff
@@ -40,6 +41,7 @@ const fields: Fields<FieldComponent> = {
   // polygon
   // 3d-model
   clipping: { Component: Clipping, hasUI: false },
+  buildingFilter: { Component: BuildingFilter, hasUI: false },
```

Comment:
This should be hasUI: true. 
Please check the design again. The ui on the right is the settings UI (in this case, it is empty), on the left is the user facing UI.
The settings UI for this component and a few others removes the arrow icon and border bottom. 

I'll do that in a different PR, so please just make it so the UI you have made in this PR is shown not inEditor, but for the user

Reply:
I fixed.
[fix: hasUI to true](https://github.com/eukarya-inc/reearth-plateauview/pull/121/commits/b29d629359b648eade13622c6b926fbeb5ae18ce)